src/v2/Components/Menu/Menu.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react"
+import styled from "styled-components"
+import {
+  BorderBox,
+  BorderBoxProps,
+  Box,
+  BoxProps,
+  Flex,
+  Sans,
+  Separator,
+  Spacer,
+  boxMixin,
+} from "@artsy/palette"
+
+const Container = styled(BorderBox)`
+  ${boxMixin}
```

Comment:
Since `BorderBox` inherits from `Box` wouldn't it already include `boxMixin`?

Reply:
Ah, yes thank you. I was combining two elements here and didn't see that.